### ISSUE TITLE
wrong WOF admin names returned in some cases

### ISSUE THREAD

**Issue by nilsnolde** [pelias/pelias#763]
#### Here's what I did :innocent:

Frankfurt in Germany seems very messy.. It confuses Frankfurt (Oder) and Frankfurt (Main) on a county level. Compare the first 3 results of:
http://pelias.github.io/compare/#/v1/search%3Ftext=frankfurt,germany&size=5

Let's take the first result as example: It returns `county` with WOF ID `102063589` and displays it as Frankfurt (Oder). However, [spelunker](https://spelunker.whosonfirst.org/id/102063589/) knows it's Frankfurt (Main).

WOF ID `102063919` is the exact other way around.

---
#### Here's what I was expecting :sparkles:

Pelias returning the same names for administrative units as WOF raw data.

---
#### Here's what I think could be improved :trophy:
Is this pointing to a more general issue? I wonder how this happens and figured the WOF importer is inserting ES documents as it finds it in the WOF records.

**Comment by orangejulius** [comment]
Hey @nilsnolde,
I just happened to take a look at this one. It appears you're right, the labels are not good here.

First, keep in mind our compare tool has a bug/limitation where the `text` property has to be last. Your query was searching for places called "Frankfurt,germany&size=5", and I think that leads to even more strange result.

Here's a URL not affected by that: http://pelias.github.io/compare/#/v1/search%3Fsize=5&text=frankfurt,germany

It shows 4 results:
```
    1) Frankfurt, Germany (locality, parented by Frankfurt (Oder))
    2) Frankfurt, Germany (locality, parented by Frankfurt am Main)
    3) Frankfurt am Main , Germany (county)
    4) Frankfurt (Oder) , Germany (county)
```

The first two records are:
- [101913837](https://spelunker.whosonfirst.org/id/101913837/)
- [101748801](https://spelunker.whosonfirst.org/id/101748801/)

Both show:

| key | value | 
| --- | ---| 
| eng_x_preferred | Frankfurt |
| eng_x_variant |  Frankfurt (`am Main` or `(Oder)`)|

Pelias uses `eng_x_preferred` over `eng_x_variant`, and I believe those values are incorrect in WOF.

@stepps00 do you think it would be appropriate to change the `eng_x_preferred` values on those records?

**Comment by stepps00** [comment]
Yeah, I think that is reasonable. We actually had some prior issues in these records (see [issue](https://github.com/whosonfirst-data/whosonfirst-data/issues/1294)), so I'm not surprised the names are slightly off.

I'll open an issue in whosonfirst-data and link to this issue, feel free to comment with what you expect to see.

**Comment by nvkelso** [comment]
@orangejulius WOF's more recent convention is to use `{lang}_x_preferred_disambiguation` for the use case of showing results in a list where the names can sometimes be confusing without some parenthetical or placetype indicator – which otherwise shouldn't be in the default / preferred name slot. 

Has Pelias adopted `{lang}_x_preferred_disambiguation` yet?

- https://github.com/whosonfirst/whosonfirst-properties/blob/71804e87f2ff33fabf9689c7b2a6238ac0317484/properties/label/README.md#lang_x_preferred_disambiguation
